src/.vuepress/components/ExtensionList.vue

```diff
@@ -47,7 +61,30 @@ export default {
 	async beforeMount() {
 		const { data } = await axios.get(GITHUB_EXTENSION_JSON);
 		const values = Object.values(groupBy(data, "lang"));
-		this.$data.extensions = sortBy(values, [(g) => this.langName(g[0].lang)]);
+		values.sort((a, b) => {
+			const langA = this.langName(a[0].lang).split(" ")[0];
+			const langB = this.langName(b[0].lang).split(" ")[0];
```

Comment:
Think this would be cleaner 
```suggestion
			const langA = ISO6391.getName(a[0].lang);
			const langB = ISO6391.getName(b[0].lang);
```

Reply:
Added it but what difference was this to the existing `langName` function?

Reply:
The thing with the existing `langName` function is that it returns the language name plus the language name in its native language with parentheses e.g. `Russian (Русский)`

The difference with the `simpleLangName` function is that it will just return the language name e.g. `Russian`

So instead of splitting the string, `Russian (Русский)`, and then getting the language name by picking the first element in the array, `Russian`, we just get the language name from `ISO6391.getName(code)` with the language code `ru` which then just returns `Russian`